### DISCUSSION TITLE
Updated to Sequoia, and not more Bookmarks

### DISCUSSION THREAD

**Discussion by pixluser** [iterate-ch/cyberduck#16589] [Q&A]
I don't know if it's a bug.
But I've updated to Sequoia, and I have no more Bookmarks.

I also switched the system language too. (to try the new mac AI).
I have no Cyberduck folder here : `~/Library/Application Support/`

And here : `/Users/XXLibrary/Group Containers/G69SCX94XU.duck/Library/Application Support/duck/Bookmarks`
I have a new bookmark that I've added (without noticing that the previous maybe weren't here).

Are they gone? Is there a way to get them back? I'm suprised that the bookmarks aren't backup into the apple cloud automatically.

**Comment by psimondo** [comment]
My bookmarks disappeared too, but then I moving my bookmarks to dropbox and putting a symlink here: /Users/XXLibrary/Group Containers/G69SCX94XU.duck/Library/Application Support/duck/Bookmarks
I created a new symlink and they're back. Perhaps you did something similar?
